puppetserver/docker-entrypoint.d/90-ca.sh

```diff
@@ -32,6 +32,9 @@ EOF
   hocon -f webserver.conf set webserver.ssl-crl-path $ssl_crl_path
   cd /
 
+  if [[ -z $CA_TTL ]]; then
+    puppet config set --section server ca_ttl "${CA_TTL}"
```

Comment:
does that check make sense? `-z` is true when the variable is empty?

Reply:
No, as CA_TTL always has a value. Fixed.